### ISSUE TITLE
Regulations page demo: Tab aren't represented using semantic headings

### ISSUE THREAD

**Issue by bsouster** [wet-boew/GCWeb#1645]
[Page in question](https://wet-boew.github.io/themes-dist/GCWeb/regulations-en.html)

The tabs provide enough context to be considered headings but are not coded as headings.

Anyone see a reason why we can't throw heading tags inside those tabs?

**Comment by GormFrank** [comment]
I agree that those tabs definitely act as headings. The way tabs work is by starting as details/summary elements, in which the `summary` element really is the "heading" of everything inside its parent `details` tag. 

That said, we are able to make them headings manually like so:

`<summary><h2 class="h6 mrgn-tp-sm mrgn-bttm-sm">About this legislation</h2></summary>`

It is now accepted as part of the HTML specifications. I added the margin classes because the headings element create unwanted space and the h6 class to make it as close to normal text as possible. @duboisp what is your take on this?
